README.md

```diff
@@ -47,13 +47,13 @@ Short on time? Check out our [5-minute tutorial ⏱️](https://tutorial.docusau
 
 Use the initialization cli to create your site:
 
-```bash
+```bash npm2yarn
```

Comment:
You probably shouldn't modify `README` 🧐 It's not used on the website

Reply:
```suggestion
```bash
```